### PR TITLE
Refactor GetClass to handler conventions.

### PR DESCRIPTION
**Pull request by krashanoff**
Another refactor PR of a GET-style endpoint. This one modifies our `GetClass` endpoint to use handler-style conventions, and adds a few tests. I'll try and make the tests more comprehensive, but this was something I just wanted to get in a draft PR at least.

I also removed our defunct class tests since I was replacing them with mocked ones.

### CODE REVIEW

File: handler/class_test.go [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+		req := httptest.NewRequest(http.MethodPost, "/?programs=true", strings.NewReader("{\"uid\": \"test\", \"cid\": \"test\"}"))
+		rec := httptest.NewRecorder()
+		assert.NotNil(t, req, rec)
+		c := echo.New().NewContext(req, rec)
+
+		if assert.NoError(t, handler.GetClass(&db.DBContext{
+			Context: c,
+			TLADB:   d,
+		})) {
+			assert.Equal(t, http.StatusOK, rec.Code)
```
Comment (lumisphere902) [inline handler/class_test.go:117]: I might be missing something but it doesn't seem like we're verifying that the programs are actually being returned here.

File: handler/class_test.go [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+			require.NotEmpty(t, rec.Body)
```
Comment (lumisphere902) [inline handler/class_test.go:142]: Same here

**Review by lumisphere902 — CHANGES_REQUESTED** [review]
Refactor LGTM. I think while we're changing these we might as well update the tests too.

**Comment by krashanoff** [comment]
100% test coverage has been achieved, the PR is ~1m old, and the functionality is there. I am going to take an executive action and say this is good to merge after a little integration testing from my end.

**Pull request merged**